dev/buildtool/spin_commands.py

```diff
@@ -46,10 +46,16 @@ class DistArch(namedtuple('DistArch', ['dist', 'arch'])):
   def __repl__(self):
     return '{},{}'.format(self.dist, self.arch)
 
+  def get_filename(self):
```

Comment:
You could shorten this by renaming the function to `filename` and annotating the function with `@property` e.g. https://github.com/spinnaker/spinnaker/blob/ae4006af07c08fc9259dd7c071bc7a39571cc6f9/dev/buildtool/base_metrics.py#L28.

Reply:
Updated